packages/googlecloud/functions/getanswer/main.py

```diff
@@ -2,23 +2,28 @@
 import time
 import math
 
-import google.cloud.logging
+# blocking google cloud for local deploy
+#import google.cloud.logging
 import functions_framework
 from supabase import create_client
-
+from dotenv import find_dotenv, load_dotenv
 from helper import parse_field, get_dbs
 from inquirer import answer_query
 import os
 import json
 
-logging_client = google.cloud.logging.Client()
-logging_client.setup_logging()
+#logging_client = google.cloud.logging.Client()
```

Comment:
commenting out gcloud functionality for local getanswer deployment - just saving it here for reference, but should be undone before full deployment